.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: daily
+    rebase-strategy: disabled
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    rebase-strategy: disabled
```

Why does the rebase strategy need to be disable in github-actions?
```suggestion
    rebase-strategy: disabled
```